src/sessions/softphone-session-handler.ts

```diff
@@ -22,22 +95,321 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
     }
   }
 
+  handleSoftphoneConversationUpdate (
+    update: ConversationUpdate,
+    participant: IConversationParticipantFromEvent,
+    callState: ICallStateFromParticipant,
+    session?: IExtendedMediaSession
+  ) {
+    const conversationId = update.id;
+    const lastConversationUpdate = this.conversations[conversationId];
+
+    if (session) {
+      session.pcParticipant = participant as any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=8384644)
```suggestion
      session.pcParticipant = participant as unknown;
```